bin.js

```diff
@@ -2,11 +2,34 @@
 
 const fs = require('fs')
 const args = require('args')
+const path = require('path')
 const pump = require('pump')
 const split = require('split2')
 const { Transform } = require('readable-stream')
 const prettyFactory = require('./')
 const CONSTANTS = require('./lib/constants')
+const { isObject } = require('./lib/utils')
+
+const bourne = require('@hapi/bourne')
+const stripJsonComments = require('strip-json-comments')
+const parseJSON = input => {
+  return bourne.parse(stripJsonComments(input), { protoAction: 'remove' })
+}
+
+const JoyCon = require('joycon')
+const joycon = new JoyCon({
+  parseJSON,
+  files: [
+    'pino-pretty.config.js',
+    '.pino-prettyrc',
+    '.pino-prettyrc.json'
+  ],
+  stopDir: path.dirname(process.cwd())
+})
+joycon.addLoader({
+  test: /\.[^.]*rc$/,
```

Comment:
Will this match files other than `pino-pretty` specific files?

Reply:
Joycon is limited to filenames listed above so we are fine there. In case custom filename is used it is actually preferable to interpret all possible rc files as json because one could invoke `pino-pretty` using `--config path/to/.customrc`.

Reply:
I really don’t understand what this is doing. There isn’t any documentation for this option that I can find.

Reply:
It is literally second example inside Joycon's usage section: https://github.com/egoist/joycon#usage

Joycon by default only recognizes filenames with `.js` & `.json` extensions. In order to instruct it to treat _dot_ rc files as json (same like babel does) you need to plug in custom loader: https://github.com/egoist/joycon#addloaderloader
Required properties are:
- `test` which is regex that filename has conform to
- `load` or `loadSync` function that reads and parses given file(path) and returns data